### PR TITLE
Feature added: getex command with args support for #275

### PR DESCRIPTION
**Pull request by apoorvyadav1111**
Addresses  #275 
  * The changes that are included in the PR.
- Support for GETEX Key command along with expiry options

  * Design document, if any.

  * Information on any implementation choices that were made.

  * Evidence of sufficient testing. You ``MUST`` indicate the tests done, either manually or automated.
 Added tests in tests/getex_test.go file

### CODE REVIEW

**Comment by JyotinderSingh** [comment]
Please resolve the conflicts by rebasing on latest master.

File: tests/getex_test.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestGetEx(t *testing.T) {
```
Comment (JyotinderSingh) [inline tests/getex_test.go:11]: Add a test case with invalid syntax (multiple expiry options specified)

**Review by JyotinderSingh — CHANGES_REQUESTED** [review]
Thanks for the changes, added a small comment.

File: tests/getex_test.go (hunk 1 above)
Comment (apoorvyadav1111) [reply tests/getex_test.go:11]: Hi @JyotinderSingh, added test cases and handled the case with multiple expiry options

**Comment by apoorvyadav1111** [comment]
Hi @JyotinderSingh, changes have been implemented. Please review.

**Review by JyotinderSingh — APPROVED** [review]
Thanks for the contribution @apoorvyadav1111! The changesook good. Approved.

**Pull request merged**